spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -109,7 +109,17 @@ object SpdxLicenseAliasMapping {
         "w3cl" to W3C,
         "wtf" to WTFPL,
         "zope" to ZPL_2_1
-    ).mapValues { (_, v) -> v.toExpression() }.let { caseSensitiveMap ->
+    ).also {
+        val keys = it.unzip().first.toMutableList()
+        val uniqueKeys = keys.distinct()
+        if (keys.size > uniqueKeys.size) {
+            uniqueKeys.forEach { uniqueKey -> keys.remove(uniqueKey) }
```

Comment:
Can't you use `keys -= uniqueKeys` or `keys.removeAll(uniqueKeys)`?

Reply:
See above.